dev_docs/cxg.md

```diff
@@ -73,38 +70,38 @@ For example:
 }
 ```
 
-### X
+### `X` array
 
-TileDB array, with a single anonymous attribute, shape (n_obs, n_var), containing the count matrix (equivalent to the AnnData `X` array).  MUST have numeric type, and SHOULD be float32.  The TileDB schema defines type and sparsity, and both dense and sparse encoding are supported.
+TileDB array, with a single anonymous attribute, shape `(n_obs, n_var)`, containing the count matrix (equivalent to the AnnData `X` array).  MUST have numeric type, and SHOULD be `float32`.  The TileDB schema defines type and sparsity, and both dense and sparse encoding are supported.
 
-### X_col_shift
+### `X_col_shift` array
 
-Optional TileDB array, used to encode-per column offsets for column-shift sparse encoding.  The TileDB array will have a single anonymous attribute, of the same type as the X array, and shape (n_var,).  
+Optional TileDB array, used to encode per-column offsets for column-shift sparse encoding.  The TileDB array will have a single anonymous attribute, of the same type as the X array, and shape `(n_var,)`.  
```

Comment:
@atolopko-czi - quick note on this -- in reading the implementation (in matrix_utils.py), I note that the code currently does _not_ correctly implement the `same type as the X array` requirement.  I think this is a code bug, and that the spec is correct.

Reply:
added bug: https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1410